src/ccache.cpp

```diff
@@ -1633,6 +1659,7 @@ calculate_result_name(Context& ctx,
     }
 
     ctx.set_manifest_name(hash_result(hash));
+    cc_log("Hashed command line: %s", ctx.hashed_args.to_string().c_str());
```

Comment:
This is incorrect since there are special cases in lines ≈1451-1561 (e.g. `-fdebug-prefix-map=`, `-MF` and `-fplugin=`) which aren't part of `ctx.hashed_args`. I would prefer removing `ctx.hashed_args` and perform tests using the regular test suite instead.

Reply:
This was me trying to convince myself that I wasn't going crazy. I forgot to remove it, sorry.